README.md

```diff
@@ -35,14 +35,22 @@ Make sure the [Requirements](README.md#requirements) are installed as well.
 
 ## Development
 
-Install [virtualenv](https://virtualenv.pypa.io/en/stable/installation/) if you haven't already.
-
 ```bash
-$ virtualenv -p python3 venv
-$ . venv/bin/activate
+# Install venv if not already installed
+$ sudo apt-get install python3-venv
+# Enter the project directory
+$ cd intermine_boot/
+# Create the Virtual environment (macOS and Linux)
+$ python3 -m venv env
+# Activate the virtual environment
+$ source env/bin/activate
+# Install packages
+$ pip install requests
```

Comment:
I'm curious what is this `requests` package for?

Reply:
Oops, that was just a silly error I made when following the python.org docs setup for venv.  I'll delete it. (Although it's a really cool http library I just learned about ;-) )

I'll delete that line.